### PR TITLE
Bump fingerprint version

### PR DESCRIPTION
**Pull request by olkononenko**
Some cache entries contain corrupted data. Instead of tracking it down and deleting those entries - let's bump the FP version.

### CODE REVIEW

File: Public/Src/Engine/Scheduler/Fingerprints/PipFingerprintingVersion.cs [hunk 1]
```diff
@@ -32,7 +32,8 @@ public enum PipFingerprintingVersion
         /// 56: Added NeedsToRunInContainer, ContainerIsolationLevel and DoubleWritePolicy
         /// 57: Fixed enumeration in StoreContentForProcessAndCreateCacheEntryAsync
         /// 58: Added RequiresAdmin field into the process pip.
+        /// 59: Bumping version due to corrupted data in the cache.
```
Comment (dannyvv) [inline Public/Src/Engine/Scheduler/Fingerprints/PipFingerprintingVersion.cs]: GitHub Test

**Comment by olkononenko** [comment]
No longer needed.

**Pull request closed without merging**